doc/source/rclcpp_cpp_client_library_overview.rst

```diff
@@ -119,92 +120,95 @@ The main entry point to the |rclcpp| API is the :cpp:class:`rclcpp::Node` class.
 The :cpp:class:`rclcpp::Node` class represents a single element in the ROS graph.
 Node's can have publishers and subscribers on topics, provide or call services, have parameters, and many other things.
 
-Creating a Node is done by calling the constructor of the Node class and providing a name for the node (after calling :cpp:func:`rclcpp::init`)::
+Creating a node is done by calling the constructor of the :cpp:class:`rclcpp::Node` class and providing a name for the node (after calling :cpp:func:`rclcpp::init`)::
 
     #include <rclcpp/rclcpp.hpp>
 
     // ...
     {
       rclcpp::init(/* ... */);
-      rclcpp::Node::SharedPtr node = rclcpp::Node::make_shared("my_node");
+      rclcpp::Node node("my_node");
```

Comment:
yeah I thought that it would be easier to digest if the simple example came first but if it's not something we encourage I'll flip the order back to how it was